packages/server/test-challenge-flow.sh

```
@@ -0,0 +1,138 @@
+const result = evalFuncAsJSON({
+  expression: expression,
+  parameters: paramObj,
+  id: 'test',
+  grid: grid
+});
+
+console.log(result.result.value);
+EOF
+
+# Solve the challenge
+cd /Users/umam3/projects/boardwalk/p-zero/dashboard/packages/server
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)